frontend/src/__tests__/cypress/cypress/utils/testConfig.ts

```diff
@@ -54,7 +54,7 @@ const AWS_PIPELINES: AWSS3Buckets = {
 
 // spread the cypressEnv variables into the cypress config
 export const cypressEnv = {
-  TEST_USER,
+  CONTRIBUTOR_USER,
```

Comment:
Same comment as above.

Reply:
Fixed in [498164e](https://github.com/opendatahub-io/odh-dashboard/pull/3469/commits/498164e80896f458e9b029d87a341a52ddc1fddf)